### PR TITLE
Spanish update 'Community' import from Crowdin

### PR DESCRIPTION
**Pull request by minimalsm**
## Description
Update Spanish with latest community changes

## Related Issue
None

### CODE REVIEW

File: src/content/translations/es/community/online/index.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Comunidades en línea
+description: Un listado de los programas de subvención entregados a través del ecosistema de Ethereum.
+sidebar: verdad
```
Comment (minimalsm) [inline src/content/translations/es/community/online/index.md:4]: Translating sidebar here

Comment (minimalsm) [inline src/content/translations/es/community/online/index.md:4]:
```suggestion
sidebar: true
```

**Comment by minimalsm** [comment]
LGTM!

**Review by wackerow — APPROVED** [review]

File: src/content/translations/es/community/online/index.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Comunidades en línea
+description: Un listado de los programas de subvención entregados a través del ecosistema de Ethereum.
+sidebar: true
+lang: en
```
Comment (wackerow) [inline src/content/translations/es/community/online/index.md:5]:
```suggestion
lang: es
```

**Pull request merged**